app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SubmissionsController < ApplicationController
+  def new
+    @submission = Submission.new
+  end
+
+  def create
+    @submission = Submission.create(submission_params)
+
+    respond_with @submission
+  end
+ 
+  def show
+    @submission = Submission.includes(:poems).find(params[:id])
```

Comment:
No. It isn't. I think I used it primarily for the sake of using it after learning about it. There's no real need for an eager load, but I figured that while the view does require the associated poems, there was no harm in it.